db/database.go

```diff
@@ -922,20 +921,23 @@ func (db *Database) Compact() (int, error) {
 			}
 		}
 
+		// Now purge them from all channel caches
+		count := len(purgedDocs)
+		purgedDocCount += count
+		if count > 0 {
+			db.changeCache.Remove(purgedDocs, startTime)
+			db.DbStats.StatsDatabase().Add(base.StatKeyNumTombstonesCompacted, int64(count))
+		}
+		base.InfofCtx(ctx, base.KeyAll, "Compacted %v tombstones", count)
```

Comment:
How about changing this to debug, and then doing info logging at the end to report total (purgedDocCount)?